packages/bits/demo/src/components/demo/paginator/paginator-test/paginator-test.component.html

```diff
@@ -0,0 +1,10 @@
+
+<div class="container">
+    <nui-paginator-basic-example></nui-paginator-basic-example>
+</div>
+
+<div>
+    <nui-paginator-adjacent-example>
+        <nui-paginator [adjacent]="2"></nui-paginator>
+    </nui-paginator-adjacent-example>
```

Comment:
Shouldn't it be outside of the other paginator?

```suggestion
    <nui-paginator-adjacent-example></nui-paginator-adjacent-example>
    <nui-paginator [adjacent]="2"></nui-paginator>
```

Reply:
Inside one was unnecessary, so I deleted it at all.